client/blocks/edit-gravatar/index.jsx

```diff
@@ -234,6 +234,11 @@ export class EditGravatar extends Component {
 							}
 						) }
 					</InfoPopover>
+					{ additionalUploadHtml && (
```

Comment:
Had to modify the original `EditGravatar` component to accomplish this.

Reply:
Yeah, not sure. Should it have children, though? I'd say this is more of a special case as what would be the children of `EditGravatar` in the logical sense?

I'll defer to your call.

Reply:
It's just at the end of the component and having it as `children` made sense to me. It also makes the component more generic and less specific, and also, you're using it as a similar syntax as children already, just with a different prop name 😉 . Happy to use `children` for that.

Reply:
Great, will do. 👍